build.gradle

```diff
@@ -130,7 +130,7 @@ subprojects {
 
 
   ext {
-    intellijVersion = '2019.3'
+    intellijVersion = '2020.1'
```

Comment:
Maybe drop a comment here for future that frontendUi/build.gradle and scripts/enforce-binary-compatibility also need to be updated when bumping `intellijVersion`